template.tpl

```diff
@@ -943,10 +943,11 @@ ___TEMPLATE_PARAMETERS___
     "name": "bingEventType",
     "displayName": "Track Type",
     "macrosInSelect": false,
+    "notSetText": "-",
     "selectItems": [
       {
-        "value": "PAGE_LOAD",
-        "displayValue": "page view"
+        "value": "CUSTOM_PAGE_LOAD",
```

Comment:
That's a good callout. Let me try this out.

Reply:
I tested this out by setting up the old bing page view tag and then swapping out the template code. It seems like the old tag does not fire a track event, which is the expected and correct behavior here.
TLDR; I think we're fine.

Reply:
See my latest commit -

Here's what was happening before my commit, for the deprecated PAGE_LOAD:
* We were submitting a track call to the SDK, with eventname === undefined
* Though this didn't result in a track call sent over the wire, as you observed, I didn't think it was clean, and if the SDK were ever to change how it handled undefined event names, could result in unwanted behavior or console messages

After my commit:
* For PAGE_LOAD, we don't submit track to SDK
* Also added a check for non-truthy eventName in the track() function, since we've had reports of errors when eventName was null

Reply:
Cool! That makes sense. Did you see the event with the undefined name in the network tab or somewhere else? We checked that the ingest worked correctly but did not check the network tab which was a miss on our part.

Reply:
did not show on network tab (track call not sent over the wire, apparently because passed eventName was undefined).

but it was being submitted to the SDK, which this bypasses.

Reply:
For our understanding for testing in future, what did you do to observe it was submitted to the SDK?

Reply:
two ways:
1. By inspection of the code flow, you could see it through to track
2. I added temporary log() statements to see that it made it thru sdk call